### DISCUSSION TITLE
Use one option for controlling debug symbols

### DISCUSSION THREAD

**Discussion by ghost** [godotengine/godot-proposals#9318] [General Discussions]
It seems a bit strange to me that we have two separate options `debug_symbols` and `separate_debug_symbols` both with `yes/no` values. I guess it was added for `compatibility` but.... Why not to have one option? `debug_symbols` with possible values: `none/separate/(embedded|builtin)`.

Does it have much sense? No, but it is strange.

**Comment by Calinou** [comment]
Thanks for the proposal! Consolidating in https://github.com/godotengine/godot/issues/72247.
